src/python/pants/util/strutil.py

```diff
@@ -141,7 +141,7 @@ def strip_v2_chroot_path(v: bytes | str) -> str:
     """
     if isinstance(v, bytes):
         v = v.decode()
-    return re.sub(r"/.*/process-execution[a-zA-Z0-9]+/", "", v)
+    return re.sub(r"/.*/pants-pe[a-zA-Z0-9]+/", "", v)
```

Comment:
How about `pants-process-<random>`? I don't think most people will know what `pe` means.

Reply:
At the risk of getting lengthy `pants-process-execution` both 1) lists "pants" and 2) matches the `-no-process-execution-local-cleanup` flag.

Thoughts?

Reply:
Switched to `pants-process-execution`